Readme changes (#35)
openvinotoolkit/mediapipe: README.md

# OpenVINO&trade; Model Server fork of [MediaPipe](https://google.github.io/mediapipe/).

> This repository is allowing users to take advantage of OpenVINO&trade; Model Serving in mediapipe examples. During the graph execution the inference calls on pretrained models are directed to model server instance initialized and parametrized by the specific inference calculators.

Reviewer: ```suggestion
This repository is allowing users to take advantage of OpenVINO&trade; Model Serving in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to model server instance initialized and parametrized by the specific inference calculators.
```
We just show in several demos full required configuration but any graph with proper adjustements should be able to run with OVMS.

Author: Done.

# OpenVINO&trade; Model Server fork of [MediaPipe](https://google.github.io/mediapipe/).

> This repository is allowing users to take advantage of OpenVINO&trade; in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to OpenVINO&trade; Model Server instance initialized and parametrized by the specific inference calculators.

Reviewer 2: ```suggestion
This repository allows users to take advantage of OpenVINO&trade; in mediapipe framework. During the graph execution the inference calls on pretrained models are directed to OpenVINO&trade; Model Server instance initialized and parametrized by the specific inference calculators.
```

Author: Done.

> We show in several demos the full required configuration but any graph with proper adjustements can run with OpenVINO&trade; backend.

Reviewer 2: ```suggestion
This repository shows multiple demos and its full configurations. It is also possible to create or adjust your own graphs to work with OpenVINO&trade; backend.
```

Author: Done.

# List of main features

> - inference calculators that allow the use of OpenVINO&trade; backend for effective inference inside mediapipe framework.

Reviewer 2: ```suggestion
- inference calculators allowing the use of OpenVINO&trade; backend for effective inference inside mediapipe framework.
```

Author: Removed.

- build your application graph file based on existing examples or (object_detection)[mediapipe/graphs/object_detection/object_detection_desktop_ovms1_graph.pbtxt] making sure that OpenVINOModelServerSessionCalculator and OpenVINOInferenceCalculator are properly placed in the inference pipeline of your graph.
- build your new application using the bazel command inside the mediapipe_ovms container, for example in object_detection case it is:
'build -c opt --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/object_detection:object_detection_openvino'
- execute your application with the specific input parameters, for example in object_detection case those are:
'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
- in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.

> Below you can find the oryginal Mediapipe repository description.

Reviewer: ```suggestion
Below you can find the original Mediapipe repository description.
```

Author: Done.
